### PR TITLE
Yamal run python implementation

### PR DESCRIPTION
**Pull request by fm-andres-rangel**

### CODE REVIEW

**Comment by fm-andres-rangel** [comment]
https://featuremine.atlassian.net/browse/FM-739

**Review by maxim-trokhimtchouk — APPROVED** [review]

File: yamal/yamal-run.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import argparse
+import signal
+import json
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config", help="Configuration path", required=True)
+    parser.add_argument("--module", help="Module name", required=False, default=None)
+    parser.add_argument("--component", help="Component name", required=False, default=None)
+    parser.add_argument("--sched", help="Run reactor in scheduled mode", action='store_true')
```
Comment (maxim-trokhimtchouk) [inline yamal/yamal-run.py:27]: should be part of config

**Review by maxim-trokhimtchouk — CHANGES_REQUESTED** [review]
make yamal-run capabilities into a function.
reactor.deploy

File: yamal/yamal-run.py (hunk 1 above)
Comment (maxim-trokhimtchouk) [reply yamal/yamal-run.py:27]: Add a test in yamal.tests for reactor.deploy

File: scripts/yamal-run.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    if (args.module is not None and args.component is None) or \
+       (args.module is None and args.component is not None):
+       raise argparse.ArgumentError("Invalid combination of arguments. module and component must either be provided through args or config.")
+
+    r = reactor()
+
+    r.deploy(args.config, args.module, args.component)
+
+    def signal_handler(sig, frame):
+        r.stop()
```
Comment (maxim-trokhimtchouk) [inline scripts/yamal-run.py:38]: move that to deploy functionality

File: scripts/yamal-run.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    parser.add_argument("--priority", help="Process CPU priority", required=False, default=None, type=int)
+    args = parser.parse_args()
+
+    if (args.module is not None and args.component is None) or \
+       (args.module is None and args.component is not None):
+       raise argparse.ArgumentError("Invalid combination of arguments. module and component must either be provided through args or config.")
+
+    r = reactor()
+
+    r.deploy(args.config, args.module, args.component)
```
Comment (maxim-trokhimtchouk) [inline scripts/yamal-run.py:35]: deploy is always for config

**Review by maxim-trokhimtchouk — CHANGES_REQUESTED** [review]

**Pull request merged**